cartographer_ros/configuration_files/backpack_3d.lua

```diff
@@ -18,6 +18,14 @@ include "sparse_pose_graph.lua"
 options = {
   sparse_pose_graph = SPARSE_POSE_GRAPH,
   trajectory_builder = TRAJECTORY_BUILDER_3D,
+  map_frame = "map",
+  odom_frame = "odom",
+  tracking_frame = "base_link",
+  provide_odom = true,
+  laser_min_range_m = 0.,
```

Comment:
We don't specify units anywhere else unless it's non-obvious. Let's prefer consistency.